File: PlacementCellBackend/Services/MentorMenteeService/Interface/IMentorMenteeService.cs
using PlacementCellBackend.DTOs.MentorMeetingDTOs;

namespace PlacementCellBackend.Services.MentorMenteeService.Interface;

public interface IMentorMenteeService
{
    Task<MeetingResponse?> CreateMentorMenteeMeeting(CreateMeetingRequest request);
    Task<bool> DeleteMentorMenteeMeeting(string eventId);
    Task<bool> UpdateMentorMenteeMeeting(UpdateMeetingRequest request);
}

File: PlacementicsBackend/src/main/java/com/example/placementicsbackend/services/mentormeentee/interfaces/IMentorMenteeService.java
package com.example.placementicsbackend.services.mentormeentee.interfaces;

import com.example.placementicsbackend.dtos.MentorMeetingDTOs.MeetingResponse;
import com.example.placementicsbackend.dtos.MentorMeetingDTOs.UpdateMeetingRequest;

public interface IMentorMenteeService {

    MeetingResponse createMentorMenteeMeeting(
            String title,
            String description,
            java.time.LocalDateTime startTime,
            java.time.LocalDateTime endTime,
            String studentEmail,
            String alumniEmail);

    boolean updateMentorMenteeMeeting(UpdateMeetingRequest request);

    boolean deleteMentorMenteeMeeting(String eventId);
}
